### DISCUSSION TITLE
DDD Constants

### DISCUSSION THREAD

**Discussion by tve** [aderusha/DDD-Printable-Wall-Control-System#76] [General]
I'm trying to make an OnShape parametric model for spacers so I can design my own tool hangers and I have a couple of questions about the constants in the design (awesome stuff, thanks so much for creating it all!). As far as I can tell it pretty much all derives from the 25.4mm pitch of the wall control slots, the 2.2mm thickness of the hooks that go into the slots, the 10x8x4 pins, and 0.1mm clearance around all perimeters that interface with other parts.

Questions:
- Any other constants I'm missing?
- What is the thickness of a flush spacer or sidepiece arm height or sidepiece back, it measures at 8.7mm in sketchup. Where does that come from?
- Do I measure correctly that the holes in the perforated spacers are 25.4mm and 12.6mm in diameter?

Thanks!

Edit: this is what I have so far, I believe all dimensions I've seen so far can be derived from this:
![image](https://github.com/aderusha/DDD-Printable-Wall-Control-System/assets/39480/2147f661-506d-4b5c-a1d0-c3821c25bde0)

**Comment by zmrow** [comment]
I would be super interested in this model if you ended up building it!
